website/docs/guide-chapter-instructions.rst

```diff
@@ -488,20 +491,29 @@ two alleles case
 
    W_{B/A} = \left[ {\frac{\sum_{i=1}^{I} {\sum_{j=1}^{J}{D_{ij}^2 / p_i } } }{ 1 - F_B }} \right]^{\frac{1}{2}} 
    
-For each locus pair the log-likelihood of obtaining the observed data
-given the inferred haplotype frequencies [``ln(L_1)``], and the
-likelihood of the data under the null hypothesis of linkage
-equilibrium [``ln(L_0)``] are given. The statistic :math:`S` (``S`` in
-the output) is defined as twice the difference between these
-likelihoods. :math:`S` has an asymptotic chi-square distribution, but the
-null distribution of :math:`S` is better approximated using a
-randomization procedure. The empirical distribution of :math:`S` is
-generated by shuffling genotypes among individuals, separately for
-each locus, thus creating linkage equilibrium. ( ``# permu`` indicates
-how many permutations were carried out). The :math:`p`-value is the
-fraction of permutations that results in values of `S` greater or
-equal to that observed. A :math:`p < 0.05` is indicative of overall
-significant LD.
+In addition to the LD measures described above, for each locus pair,
+we describe three additional measures related to the log-likelihood
+that are displayed in the output above:
+
+:math:`\ln(L_1)`
+   the log-likelihood of obtaining the observed data given the inferred
+   haplotype frequencies (``ln(L_1)`` in the output)
+
+:math:`\ln(L_0)`   
+   the log-likelihood of the data under the null hypothesis of linkage
+   equilibrium (``ln(L_0)`` in the output)
+
+:math:`S`
+   the statistic (``S`` in the output) is defined as twice the
+   difference between these likelihoods. :math:`S` has an asymptotic
+   chi-square distribution, but the null distribution of :math:`S` is
+   better approximated using a randomization procedure. The empirical
+   distribution of :math:`S` is generated by shuffling genotypes among
+   individuals, separately for each locus, thus creating linkage
+   equilibrium. ( ``# permu`` indicates how many permutations were
+   carried out). The :math:`p`-value is the fraction of permutations
+   that results in values of `S` greater or equal to that observed. A
+   :math:`p < 0.05` is indicative of overall significant LD.
```

Comment:
@rsingle  should we keep/modify this?

Reply:
I think adding "If a permutation test is requested (allPairwiseLDWithPermu=1)," at the beginning of the following sentence under the 'S' subsection: "The empirical distribution of is generated by shuffling ... "
would take care of it.

Reply:
@rsingle how do the rest of the changes look? You can click the submit review button and approve if they look good. Then I'll add this last one after that.

Reply:
> I think adding "If a permutation test is requested (allPairwiseLDWithPermu=1)," at the beginning of the following sentence under the 'S' subsection: "The empirical distribution of is generated by shuffling ... " would take care of it.

Agreed.  I added that in the latest commit.  Also note that `p-value` also only appears if `allPairwiseLDWithPermu` is set to non-zero.